build/rollup.js

```diff
@@ -49,7 +49,7 @@ export default [
   defaults({
     input: 'src/index.ts',
     output: [
-      {file: 'dist/index.cjs.js', format: 'cjs'},
+      {file: 'dist/index.cjs', format: 'cjs', exports: 'named'},
```

Comment:
@robbtraister I don't understand why you have to remove `.js`? 😅

Reply:
I just had this error when I reverted:
> ⠙ (node:44675) Warning: require() of ES modules is not supported.
require() of /Users/lucaban/.ghq/github.com/mesqueeb/merge-anything/dist/index.cjs.js from /Users/lucaban/.ghq/github.com/mesqueeb/merge-anything/test/index.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename index.cjs.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /Users/lucaban/.ghq/github.com/mesqueeb/merge-anything/package.json.

Reply:
https://nodejs.org/api/esm.html#esm_package_scope_and_file_extensions